spec/features/merchants/invoices/index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe "Merchant Invoices Index", type: :feature do
+  # As a merchant,
+  # When I visit my merchant's invoices index (/merchants/:merchant_id/invoices)
+  # Then I see all of the invoices that include at least one of my merchant's items
+  # And for each invoice I see its id
+  # And each id links to the merchant invoice show page
+  describe "When a user visits merchant's invoices index, there is information" do
+    xit "They see all invoices with at least one of merchant's items" do
+      visit "/merchants/#{@merchant_1.id}/invoices"
```

Comment:
we should be using the `merchant_invoices_path(@merchant_1)` helper

Reply:
I have added the resources you sent into the routes file and will continue using the helpers from here on out.